mkdocs.yml

```diff
@@ -30,7 +30,7 @@ nav:
     - Contributing to the Docs: contributing-docs.md
     - Related Projects: related-projects.md
     - Development Tips: development-tips.md
-    - How We Use Travis CI: travis.md
+    - How We Use GitHub Actions: github_actions.md
```

Comment:
Is there a `github_actions.md`?

Reply:
There... should be. It might've gotten lost in all my branch cleanup.